applicationconfig/client/src/main/java/com/azure/applicationconfig/models/KeyValueFilter.java

```diff
@@ -19,7 +19,7 @@ public KeyValueFilter withIfMatch(String etag) {
      * @return 'If-Match' header value to be added
      */
     public String ifMatch() {
-        return ifMatch;
+        return ifMatch == null ? null : "\"" + ifMatch + "\"";
```

Comment:
Rather than return null it is more API-friendly to return an empty string (unless null is special whereas an empty string is not). Same for below.